src/OSPSuite.Presentation/Presenters/ParameterIdentifications/ParameterIdentificationPredictedVsObservedChartPresenter.cs

```diff
@@ -83,6 +87,15 @@ private void plotAllCalculations(DataRepository simulationResult, Action<DataCol
          var calculationColumns = calculationColumnsToPlot(simulationResult);
          calculationColumns.Each(x => plotCalculationColumn(x, action));
       }
+      private void addDeviationLines(float foldValue)
+      {
+         var observationColumns = _parameterIdentification.AllObservedData.Select(x => x.FirstDataColumn()).ToList();
+         _deviationLineRepositories.AddRange(_predictedVsObservedChartService.AddDeviationLine(foldValue, observationColumns, Chart, _deviationLineRepositories.Count/2));
```

Comment:
That i don't undertand at all. What's the count divided by 2 etc.
Stuff like this needs to be commented in code ad this is not obvious at all (at least to me)

Reply:
Should not be devided by two, it is wrong that it is there. removed.